### PR TITLE
Run unit tests in Windows CI builds

### PR DESCRIPTION
**Pull request by bejado**

### CODE REVIEW

File: libs/image/tests/test_image.cpp [hunk 1]
```diff
@@ -388,7 +388,7 @@ TEST_F(ImageTest, Ktx) { // NOLINT
         const auto fileSize = getFileSize(path.c_str());
         ASSERT_GT(fileSize, 0);
         vector<uint8_t> buffer(fileSize);
-        std::ifstream in(path, std::ifstream::in);
+        std::ifstream in(path, std::ifstream::in | std::ifstream::binary);
```
Comment (bejado) [inline libs/image/tests/test_image.cpp]: This fixes another Windows unit test issue

Comment (jaibeer72) [reply libs/image/tests/test_image.cpp]: Should it not by default assume it was meant to be binary ? or have I missed something?

Comment (prideout) [reply libs/image/tests/test_image.cpp]: There is likely a \r\n sequence somewhere in the KTX file, which would get mistakenly coalesced on Windows.  This is a good fix.

Comment (romainguy) [reply libs/image/tests/test_image.cpp]: We had the same issue in other places.

Comment (bejado) [reply libs/image/tests/test_image.cpp]: I put this PR on hold until after we move to GitHub Actions, but I can merge just this fix now.

**Pull request closed without merging**